src/contents/articles/hello-world/index.md

```diff
@@ -27,4 +25,4 @@ We want to grow as software developers but we also want to grow as a people. As
 
```

Comment:
The section - **Why do we need a community?** doesn't answer the question the heading posed. I think this may be a good opportunity for you to speak about your experiences in the Edinburgh developer community. In that way your rationale for why a community should exist will be based on evidence rather than hypotheses, which is more convincing.